app/dashboard/templates/shared/rate_bounties.html

```diff
@@ -46,8 +46,8 @@
                   {% else %}
                     <img class="card-rating__avatar rounded-circle" alt="{{ bounty.bounty_owner_github_username }}" width="56" height="56" src="{% url 'org_avatar' bounty.bounty_owner_github_username %}">
                   {% endif %}
-                  <div class="ml-2 text-dark">
-                    <a href="{{ bounty.url }}" class="card-rating__title">{{ bounty.title }}</a>
+                  <div class="ml-2 {% if is_my_profile and profile.custom_theme == 'light' %} text-dark {% elif is_my_profile and profile.custom_theme == 'dark' %} text-light {% else %} text-light {% endif %}">
```

Comment:
seems that it'd be maintainable/easier from a DOM perspective to just have a `.theme-dark` class that you attach to the `<body>` element.. rather than having to special case all the classes across the DOM.